src/Hl7.Fhir.Specification.Shared.Tests/Hl7.Fhir.Specification.Shared.Tests.projitems

```diff
@@ -21,7 +21,6 @@
     <Compile Include="$(MSBuildThisFileDirectory)Snapshot\SnapshotElementMatcherTests.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Snapshot\SnapshotGeneratorManifestTests.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Snapshot\SnapshotGeneratorTest.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Snapshot\SnapshotGeneratorTypeSlicingTests.cs" />
```

Comment:
Why is this gone?

Reply:
There is an answer, right above your question???

Reply:
I can see them now :)

And I approve